src/couch/src/couch.erl

```diff
@@ -13,6 +13,7 @@
 -module(couch).
 
 -compile(export_all).
+-compile(nowarn_export_all).
```

Comment:
I would explicitly export only things we really need:
- start/0
- stop/0
- restart/0
- start_apps/1 (hmm I didn't know about it. We have something similar in test_util.)

Reply:
Agreed, although it seems like `start_apps/1` should maybe not be exported, since I'm not finding it in use anywhere else.